about_proxy_object_project.rb

```diff
@@ -15,10 +15,29 @@
 class Proxy
   def initialize(target_object)
     @object = target_object
-    # ADD MORE CODE HERE
+    @messages = {} # key is a method name, value is a number of times called
   end
 
-  # WRITE CODE HERE
+    def method_missing(method_name, *args, &block)
```

Comment:
please fix alignment

Reply:
Thanks. Corrected in the last commit "corrected task".

Reply:
looks like you didn't make changes